### ISSUE TITLE
follow people

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#105]
```
If someone mentions a user I don't follow or RTs him, I'd like to right
click that user's name and hit 'Follow', or something similar.
```

Original issue reported on code.google.com by `subway.c...@gmail.com` on 26 Feb 2010 at 1:52

**Comment by GoogleCodeExporter** [comment]
```
its a essencial function for sure!
```

Original comment by `stuffm...@gmail.com` on 1 Mar 2010 at 2:42

**Comment by GoogleCodeExporter** [comment]
```
Closed by 81ddd2c44b revision.
```

Original comment by `tro...@gmail.com` on 4 Mar 2010 at 2:36
- Changed state: **Fixed**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**
